app-arch/gzip/gzip-1.10-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+DESCRIPTION="Standard GNU compressor"
+HOMEPAGE="https://www.gnu.org/software/gzip/"
+SRC_URI="mirror://gnu/gzip/${P}.tar.xz
+	mirror://gnu-alpha/gzip/${P}.tar.xz
+	mirror://gentoo/${P}.tar.xz"
```

Comment:
Is the mirror://gentoo a historical artefact here?

Reply:
In the end it is, yes. Will get rid of it.